docs/src/pages/docs/installation/kubernetes.mdx

```diff
@@ -94,25 +94,21 @@ postgresql:
 
 #### Dependencies
 
-You can specify pip packages to be installed before startup, e.g. to install extra database drivers:
+You can specify pip packages to be installed before startup. The following example installs the Big Query and Elasticsearch database drivers so that you can connect to those datasources in your Superset installation.
 
 ```yaml
-additionalRequirements:
```

Comment:
`additionalRequiremnts` are no longer supported value?

Reply:
no this broke, please see #14188 and #13917. This PR may be a moot point if this chart is released up, see https://github.com/apache/superset/issues/14188#issuecomment-903883464 which is a comment from 14 days ago.